AUDITING.md

```diff
@@ -0,0 +1,75 @@
+# Auditing
+
+This document defines how Platform Chaos audits chaos events. It lists what is tracked and how it is formatted.
+
+## Tracked Data
+
+When a chaos event is terminated through an *API*, *CLI*, or *Extension*, the following properties are recorded and exported through a JSON object.
+  - An `id` unique to this event instantiation (`guid`). 
+  - A `chaos-type` of enumerable type `initiate`
+  - `user` and `system` that terminated the event
+  - `date` of event initiation (including time `UTC`)
+  - targeted `resources`. This is broken down by `subscriptionID` and then by `resourceGroup`
+  - Chaos `event` information (`eventID`, `eventName`, `eventURL`)
+
+Additionally an *Extension* may record additional information regarding the specificities of the event. This information is stored within an `extension` property.
+
+Every chaos event initiation/termination will be logged. The general structure of an event log is defined below:
+
+- _event log_ (Object)
+  - `id` (String) 
+    - GUID unique to this event instantiation
+  - `type` (String) 
+    - One of `'initiate'`, `'terminate'`
+  - `user` (String)
+    - Some identifiable piece of information specifying which user executed the chaos event
+  - `system` (String | Object)
+    - Some identifiable piece of information specifying which system the chaos event was executed from
+    - If more system information can be provided, do so in the form of an object
+  - `date` (String)
+    - UTC timestamp of chaos event initiation
+  - `resources` (Object)
```

Comment:
why did we move away from the existing definition of a target/resource for a chaos event? I'd much prefer to keep the same structure here, rather then invent a new one. However, if we do need a new one let's try to use terminology and structure that isn't azure specific as we aim to keep the guts of our system cloud generic.

Reply:
I did this in order to support glob patterns in the target/resource thing. For the first version of this should I just specify: 
```
{
  "resource": "subId/resGrpId/resName",
}

Reply:
yes. lets not introduce a new concept here if we can avoid it.